flang/test/Lower/Intrinsics/bge.f90

```diff
@@ -156,3 +156,20 @@ subroutine bge_test11(c)
   ! CHECK: %[[V:.*]] = fir.convert %[[R]] : (i1) -> !fir.logical<4>
   ! CHECK: fir.store %[[V]] to %[[C]] : !fir.ref<!fir.logical<4>>
 end subroutine bge_test11
+
+! CHECK-LABEL: bge_test12
```

Comment:
Nit: don't you need to add the -fwrap flag to the RUN line for test to be relevant?

Also, to avoid updating the legacy lowering code just for the tests. I would advise adding all the new tests in a new `test/Lower/frwap-option.f90` file run with HFLIR.

I think you can use CHECK-NOT here to too make the tests smaller.

Reply:
Thank you for your advice. I'll create another test file and add `-fwrapv` to the `RUN` command.